blocks/author-profile/author-profile.css

```diff
@@ -0,0 +1,70 @@
+.author-profile {
+  padding-top: var(--udexSpacer20);
+}
+
+.author-profile > div {
+  display: flex;
+  flex-direction: row;
+  padding: var(--udexSpacer24) var(--udexSpacer20);
+  gap: var(--udexSpacer24);
+  border: 1px solid var(--udexColorGrey2);
+  box-shadow: 0 2px 4px rgb(34 54 73 / 20%);
```

Comment:
I guess you''ve asked yourself the obvious - is there a var for this? - question already?

Reply:
Yes, but I haven't found one. Will update in a separate issue once I have design token